Workbooks/Citrix_ADC_WAF.json

```diff
@@ -0,0 +1,42 @@
+{
+  "version": "Notebook/1.0",
```

Comment:
1. Please reflect the DeviceProduct name changes to the queries in this workbook as well.
2. Also this workbook looks lean. Is it possible to provide a few more visuals on the data, like trend of SQL injections and other vulnerabilities etc.?